config/projects.yml

```diff
@@ -182,7 +182,7 @@ doctrine-phpcr-admin-bundle:
   branches:
     master:
       # sonata 2 should still support php 5.6 to be consistent with the symfony cmf
-      php: ['5.6', '7.0', '7.1']
+      php: ['5.6', '7.1']
```

Comment:
we are about to release 2.0, so i think its too early to drop 7.0. also dropping 7.0 but not 5.6 would be rather pointless. i suggest we wait until we tagged the stable release and created a 2.x branch and only then drop 5.6 and 7.0 for master that will be 3.x